templates/taxi/manufacturer_list.html

```diff
@@ -1,32 +1,31 @@
 {% extends "base.html" %}
 
 {% block content %}
-    <h1>Manufacturer List
-    </h1>
+    <h1>Manufacturer List</h1>
+
+    <a href="{% url 'taxi:manufacturer-create' %}">Create new Manufacturer</a>
 
     {% if manufacturer_list %}
       <table class="table">
         <tr>
           <th>ID</th>
           <th>Name</th>
           <th>Country</th>
+          <th>Actions</th>
         </tr>
 
         {% for manufacturer in manufacturer_list %}
             <tr>
+              <td>{{ manufacturer.id }}</td>
+              <td>{{ manufacturer.name }}</td>
+              <td>{{ manufacturer.country }}</td>
               <td>
-                  {{ manufacturer.id }}
-              </td>
-              <td>
-                  {{ manufacturer.name }}
-              </td>
-              <td>
-                  {{ manufacturer.country }}
+                  <a href="{% url 'taxi:manufacturer-update' manufacturer.id %}">Update</a> |
+                  <a href="{% url 'taxi:manufacturer-delete' manufacturer.id %}">Delete</a>
```

Comment:
As per the checklist, it would be better to add a `Cancel` button that leads to the previous page the user was on. This will improve the user experience by providing an option to cancel the delete operation.